src/components/notes/saving-indicator.styl

```diff
@@ -0,0 +1,12 @@
+@require 'settings/breakpoints.styl'
+
+.saving-indicator
+    position absolute
+    bottom 1rem
+    right 1rem
+    padding 0.25rem 0.75rem
+    color var(--cool-grey, #95999d)
+    background-color: #FFFFFFAA
```

Comment:
```suggestion
    background-color: #FFFFFF
```

Reply:
Nope, I really want the alpha transparency

Reply:
Almost ashamed to say this, but TIL. Always thought the only way to get transparency was to use `rgba`!